src/utils/vevent.js

```diff
@@ -0,0 +1,15 @@
+import { nanoid } from 'nanoid';
+
+export default class Vevent {
+
+    constructor(start, end) {
+
+       this.productId = '-//St. Isidore Guild//Ember Days App//EN';
+       this.title = 'Ember Day';
+       this.description = "Prayer and Fasting";
+       this.start = start;
+       this.end = end;
```

Comment:
can calculating an end date be replaced with `this.duration = { days: 1 }`?

Reply:
good catch. much easier. done.